app/presenters/document_export_presenter.rb

```diff
@@ -55,4 +58,22 @@ def public_url_for_admin_link(url)
     edition = Whitehall::AdminLinkLookup.find_edition(url)
     Whitehall.url_maker.public_document_url(edition) if edition.present?
   end
+
+  def political_info(edition)
+    {
+      political: edition.political?,
+      government: government(edition)
+    }
+  end
+
+  def government(edition)
+    gov = edition.government
+    return nil unless gov
+
+    {
+      title: gov.name,
+      slug: gov.slug,
+      current: gov.current?
```

Comment:
Instead of a boolean for this being the current government, would the start/end dates of the government be a better value to include?  This would then allow Content Publisher to determine itself whether the government is current.  Since we haven't yet implemented history mode in Content Publisher, it might be better to have more data than less.

With that in mind, would it be possible just to include the entire `Government` object here since it contains all this data?

Reply:
Yes sure I'll include the full `Government` object, I initially went with this approach as it's all publishing-api needs to represent the document as political, but more data is better than less so... changes incoming 💯